pipeline_tools/tests/shared/submission/test_confirm_submission.py

```diff
@@ -68,50 +82,83 @@ def _request_callback(request, context):
             confirm_submission.wait_for_valid_status(envelope_url, HttpRequests())
         assert requests_mock.call_count == 3
 
-    def test_confirm_success(self, requests_mock):
+    def test_confirm_success(self, requests_mock, test_data):
         envelope_url = (
             'https://api.ingest.dev.data.humancellatlas.org/submissionEnvelopes/abcde'
         )
 
         def _request_callback(request, context):
             context.status_code = 200
+            assert request.headers.get('Authorization') == test_data.headers.get(
+                'Authorization'
+            )
             return {}
 
         requests_mock.put(
             '{}/submissionEvent'.format(envelope_url), json=_request_callback
         )
-        with HttpRequestsManager():
-            confirm_submission.confirm(envelope_url, HttpRequests())
+        with HttpRequestsManager(), mock.patch(
+            'pipeline_tools.shared.auth_utils.DCPAuthClient.get_auth_header',
+            return_value=test_data.headers,
+        ):
+            confirm_submission.confirm(
+                envelope_url,
+                HttpRequests(),
+                test_data.runtime_environment,
+                test_data.service_account_path,
+            )
         assert requests_mock.call_count == 1
 
-    def test_confirm_retries_on_error(self, requests_mock):
+    def test_confirm_retries_on_error(self, requests_mock, test_data):
         envelope_url = (
             'https://api.ingest.dev.data.humancellatlas.org/submissionEnvelopes/abcde'
         )
 
         def _request_callback(request, context):
             context.status_code = 500
+            assert request.headers.get('Authorization') == test_data.headers.get(
```

Comment:
Since the test is mocking the return value of `DCPAuthClient.get_auth_header` I think this will always be true so I don't think you have to assert it here